### ISSUE TITLE
Sprites displaying incorrectly in editor (dev)

### ISSUE THREAD

**Issue by kosiecki17** [LLK/scratch-gui#4109]
While viewing sprites in dev: previews in the library are correct, but when the sprite loads in the stage, it is missing pieces and has other pieces that are displaying incorrectly. Costumes appear normal. When you move a piece of the sprite in the paint editor, they update and display correctly on the stage. All of the additional costumes are displaying correctly.

So far I have only found 2 sprites with this specific issue: Andie and Bat.

<img width="1069" alt="screen shot 2018-12-17 at 2 22 56 pm" src="https://user-images.githubusercontent.com/25378759/50111024-bec42e80-0209-11e9-9af0-68303fddeeec.png">
In the case of the bat sprite, the ears are displaying incorrectly in both places, as well as in all of the additional costumes.
<img width="488" alt="screen shot 2018-12-17 at 2 41 53 pm" src="https://user-images.githubusercontent.com/25378759/50111112-00ed7000-020a-11e9-84aa-ec866eb0a2ad.png">

<img width="698" alt="screen shot 2018-12-17 at 2 28 30 pm" src="https://user-images.githubusercontent.com/25378759/50111030-c388e280-0209-11e9-9f1e-b4dc3ace2952.png">

In the case of Andie, the missing pieces aren't all separate vectors in the SVG.
<img width="1052" alt="screen shot 2018-12-17 at 2 33 11 pm" src="https://user-images.githubusercontent.com/25378759/50111580-11521a80-020b-11e9-8bac-b5b3354f2e0b.png">

_Mac OS 10.13.6 Chrome 71.0.3578.80_

**Comment by fsih** [comment]
Paul wrote a script to compare svgs before and after treatment with SVGRenderer which is super helpful, thanks Paul! [costumes-before-after-v2-svg-renderer.html.zip](https://github.com/LLK/scratch-gui/files/2687678/costumes-before-after-v2-svg-renderer.html.zip)

Here are the broken results I found. I think I'm seeing 3 things

1. Missing pieces issue
2. Fills turning black issue
3. Strokes becoming thin

For 1 and 2, I suspect it's the recent change to SVGRenderer, https://github.com/LLK/scratch-svg-renderer/pull/62. I'll investigate.

For 3, @kosiecki17 I think the tufts on the bat becoming thin might be an issue that's been around for awhile? I'm seeing it on beta.scratch.mit.edu. I wonder if it's a custom Illustrator thing that doesn't work when you view it elsewhere? We could convert the strokes to shapes if we want to keep how it looks.

![image](https://user-images.githubusercontent.com/2855464/50112617-9b9b7e00-020d-11e9-9890-96db2ac623e1.png)
![image](https://user-images.githubusercontent.com/2855464/50112636-ace48a80-020d-11e9-80ee-4870b484dcc4.png)
![image](https://user-images.githubusercontent.com/2855464/50112641-b1a93e80-020d-11e9-85c4-0a4ed9cef84b.png)

**Comment by kosiecki17** [comment]
Those tufts on the bat's ears do not have strokes in the SVG files.
They _look_ like strokes in the paint editor, but are shapes there as well.  
I thought they were rounded strokes until I looked closer.

**Comment by fsih** [comment]
Oh no, the ones with missing pieces use masks allll over the place X|
I think we decided we weren't going to support masks, but we can't really do that if they're in our library

**Comment by fsih** [comment]
Status:
For issue 1, @kosiecki17 is working on removing masks from the bat costumes and andie-a
For issue 2, https://github.com/LLK/scratch-svg-renderer/pull/68 is up
For issue 3, we figured out that the ear tufts are supposed to be thin

**Comment by kosiecki17** [comment]
I've updated all of the costumes for the bat. The ears were a separate issue. I did not find any clues as to why the left arm and leg would not be displaying.
I've also updated Andie's first costume. 

I found no clipping masks in any of these files while viewing them in illustrator. I also opened them in a text reader and searched the SVG for the words "clipping" and "mask." If there are other ways for me to check this, let me know.
